admin/database/database.go

```diff
@@ -245,6 +245,7 @@ type UpdateProjectOptions struct {
 	ProdBranch           string
 	ProdVariables        map[string]string
 	ProdDeploymentID     *string
+	ReconcileDeployments bool
```

Comment:
This doesn't belong in the database options struct – not used in database call.

Reply:
This problem arises bcz of choice of keeping `database.UpdateProjectOptions` in service methods. Ideally service should take its own options where `database.UpdateProjectOptions` is one field. For now I will add `ReconcileDeployments` as additional method argument.